main.tf

```diff
@@ -23,36 +23,40 @@ resource "null_resource" "shell" {
   }
 
   provisioner "local-exec" {
-    command = "${local.command_chomped} 2>\"${path.module}/stderr.${null_resource.start.id}\" >\"${path.module}/stdout.${null_resource.start.id}\"; echo $? >\"${path.module}/exitstatus.${null_resource.start.id}\""
+    command     = "${local.command_chomped} 2>\"${abspath(path.module)}/stderr.${null_resource.start.id}\" >\"${abspath(path.module)}/stdout.${null_resource.start.id}\"; echo $? >\"${abspath(path.module)}/exitstatus.${null_resource.start.id}\""
+    environment = var.environment_variables
```

Comment:
I think `environment` would be a better name - it's `environment:` in docker-compose for example.

Reply:
updated